addons/knowledge/security/security.xml

```diff
@@ -18,6 +18,16 @@
         <field name="groups" eval="[(4, ref('base.group_user'))]"/>
     </record>
 
+    <record id="knowledge_user_read_articles" model="ir.rule">
```

Comment:
knowledge_**portal**_read_articles
But anyway, as this is the exact same rule than for base.users, we could just add the portal group to the user's rule.